fundamentals/streaming/README.md

```diff
@@ -4,12 +4,13 @@ This repository contains sample code for the article "Streaming Responses in ASP
 
 ## What
 
-The project contains two implementations of the definition of a simple ASP.NET Core web site that contains endpoints that sing a song:
+The project contains three implementations of the definition of a basic ASP.NET Core web site that contains endpoints that play a song:
 
 * `AsynchronousWithSystemTextJson`
 * `SynchronousWithNewtonsoftJson`
+* `DynamicBinaryStream`
 
-These endpoints sing the song indefinitely, streaming the song repeatedly until the client disconnects. This acts as a demonstration of how to implement streaming endpoints with ASP.NET Core.
+These endpoints play the song indefinitely, streaming the song repeatedly until the client disconnects. This acts as a demonstration of how to implement streaming endpoints with ASP.NET Core.
```

Comment:
I'm not sure this change makes sense. They literally do _not_ "play" the song.

Reply:
```suggestion
These endpoints stream the song indefinitely, repeating until the client disconnects. This acts as a demonstration of how to implement streaming endpoints with ASP.NET Core.
```